podcasts/End of Year/Stories/Views/PodcastCover.swift

```diff
@@ -9,21 +9,38 @@ struct PodcastCover: View {
     /// Whether this is a big cover, in which shadows should be bigger
     let big: Bool
 
+    /// The color of the view the cover will appear on
+    /// Prevents a flickering issue on dark backgrounds
+    let viewBackgroundStyle: ThemeStyle?
+
     @State private var image: UIImage?
 
-    init(podcastUuid: String, big: Bool = false) {
+    init(podcastUuid: String, big: Bool = false, viewBackgroundStyle: ThemeStyle? = nil) {
         self.podcastUuid = podcastUuid
         self.big = big
+        self.viewBackgroundStyle = viewBackgroundStyle
+    }
+
+    private var rectangleColor: Color? {
+        var rectangleColor: Color?
+
+        if let viewBackgroundStyle {
+            rectangleColor = AppTheme.color(for: viewBackgroundStyle)
+        }
+
+        return rectangleColor
     }
 
     var body: some View {
         ZStack {
             Group {
                 if big {
                     Rectangle()
+                        .foregroundColor(rectangleColor)
```

Comment:
When the color is nil the `Rectangle` will behave the same as if it was without this property?